Sources/GRPC/GRPCChannel/GRPCChannelBuilder.swift

```diff
@@ -235,6 +235,16 @@ extension ClientConnection.Builder.Secure {
     self.tls.certificateVerification = certificateVerification
     return self
   }
+
+  /// A custom verification callback that allows completely overriding the certificate verification logic.
+  @discardableResult
+  public func withCustomVerificationCallback(
```

Comment:
This should be `withTLSCustomVerificationCallback`: without `TLS` it's unclear what is being verified.